src/SonarScanner.MSBuild.PreProcessor/Protobuf/AnalysisCacheMsg.proto

```diff
@@ -0,0 +1,13 @@
+﻿/*
+  In order to update the generated file run
+  .\protoc.exe --proto_path=[sonar-scanner-msbuild-path]\src\SonarScanner.MSBuild.PreProcessor\Protobuf --csharp_out=[sonar-scanner-msbuild-path]\src\SonarScanner.MSBuild.PreProcessor\Protobuf [sonar-scanner-msbuild-path]\src\SonarScanner.MSBuild.PreProcessor\Protobuf\AnalysisCacheMsg.proto
+ */
+syntax = "proto3";
+
+option java_package = "org.sonar.scanner.protocol.internal";
+option optimize_for = SPEED;
+option csharp_namespace = "SonarScanner.MSBuild.PreProcessor.Protobuf";
+    
+message AnalysisCacheMsg {
+  map<string, bytes> map = 1;
```

Comment:
Let's say what it is to allow extensibility in the future
```suggestion
  map<string, bytes> fileHashes = 1;
```

Reply:
As mentioned above, we cannot change this contract without changing the implementation on the server too.